parter.md

```diff
@@ -0,0 +1,69 @@
+
+# Support
+
+If you need help with this integration, please contact Relay Support at [hello@relay.sh](mailto:hello@relay.sh). 
+
+# Integration Walkthrough
+
+There are two ways to integrate with PagerDuty: via Global Event Routing or on a PagerDuty Service. If you are adding this integration to an existing PagerDuty service, please skip to the Integrating with a PagerDuty Service section of this guide.
+There are two kinds of Relay integrations that can be added to a PagerDuty Service:
+- One Relay integration is the [PagerDuty Trigger](https://github.com/relay-integrations/relay-pagerduty/tree/master/triggers) and may be added by following the [Integrating Relay Triggers With a PagerDuty Service](#integrating-relay-triggers-with-a-pagerduty-service) section of this guide.
+- The other Relay integration is the [PagerDuty Steps](https://github.com/relay-integrations/relay-pagerduty/tree/master/steps) and may be added by following the [Integrating Relay Steps With a PagerDuty Service](#integrating-relay-steps-with-a-pagerduty-service) section of this guide.
+
+## In PagerDuty
+
+### Integrating Relay Steps With a PagerDuty Service
+1. From the **Configuration** menu, select **Services**.
+2. There are two ways to add an integration to a service:
+   * **If you are adding your integration to an existing service**: Click the **name** of the service you want to add the integration to. Then, select the **Integrations** tab and click the **New Integration** button.
+   * **If you are creating a new service for your integration**: Please read PagerDuty's documentation in section [Services and Integrations](https://support.pagerduty.com/docs/services-and-integrations) and follow the steps outlined in the [Create a New Service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) section, selecting Relay as the **Integration Type** in step 4. Continue with the [In  Relay](#in-relay)  section (below) once you have finished these steps.
+3. Enter an **Integration Name** such as `Relay` or `relay-by-puppet` and select Relay from the Integration Type menu.
+4. Click the **Add Integration** button to save your new integration. You will be redirected to the Integrations tab for your service.
+5. An **Integration Key** will be generated on this screen. Keep this key saved in a safe place, as it will be used when you configure the integration with Relay in the next section.
+6. Continue with the [In Relay](#in-relay) section below.
+![](media/integration-key.png)
+
+### Integrating Relay Triggers With a PagerDuty Service
+1. From the **Configuration** menu, select **Services**.
```

Comment:
The first step here should be to add the workflow to relay and expand the settings tab - otherwise they won't know where the webhook URL came from.